packages/jest-haste-map/src/constants.ts

```diff
@@ -16,7 +14,10 @@
  * This constant key map allows to keep the map smaller without having to build
  * a custom serialization library.
  */
-export default {
+
+import {HType} from './types';
+
+const constants: HType = {
```

Comment:
Explicitly typing this ensure the type remains narrow rather than expanded to `number`